frontend/src/container/MetricsApplication/Tabs/Overview/ServiceOverview.tsx

```diff
@@ -0,0 +1,118 @@
+import getServiceOverview from 'api/metrics/getServiceOverview';
+import Spinner from 'components/Spinner';
+import Graph from 'container/GridGraphLayout/Graph/';
+import { GraphTitle } from 'container/MetricsApplication/constant';
+import { getWidgetQueryBuilder } from 'container/MetricsApplication/MetricsApplication.factory';
+import { letency } from 'container/MetricsApplication/MetricsPageQueries/OverviewQueries';
+import { Card, GraphContainer } from 'container/MetricsApplication/styles';
+import useResourceAttribute from 'hooks/useResourceAttribute';
+import { convertRawQueriesToTraceSelectedTags } from 'hooks/useResourceAttribute/utils';
+import getStep from 'lib/getStep';
+import { useMemo } from 'react';
+import { useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { AppState } from 'store/reducers';
+import { PayloadProps } from 'types/api/metrics/getServiceOverview';
+import { TagFilterItem } from 'types/api/queryBuilder/queryBuilderData';
+import { EQueryType } from 'types/common/dashboard';
+import { GlobalReducer } from 'types/reducer/globalTime';
+import { Tags } from 'types/reducer/trace';
+import { v4 as uuid } from 'uuid';
+
+import { ClickHandlerType } from '../Overview';
+import { Button } from '../styles';
+import { onViewTracePopupClick } from '../util';
+
+function ServiceOverview({
+	onDragSelect,
+	handleGraphClick,
+	selectedTraceTags,
+	selectedTimeStamp,
+	tagFilterItems,
+}: ServiceOverviewProps): JSX.Element {
+	const { maxTime, minTime } = useSelector<AppState, GlobalReducer>(
+		(state) => state.globalTime,
+	);
+	const { servicename } = useParams<{ servicename?: string }>();
+	const { queries } = useResourceAttribute();
+	const selectedTags = useMemo(
+		() => (convertRawQueriesToTraceSelectedTags(queries) as Tags[]) || [],
+		[queries],
+	);
+
+	const latencyWidget = useMemo(
+		() =>
+			getWidgetQueryBuilder(
+				{
+					queryType: EQueryType.QUERY_BUILDER,
+					promql: [],
+					builder: letency({
+						servicename,
+						tagFilterItems,
+					}),
+					clickhouse_sql: [],
+					id: uuid(),
+				},
+				GraphTitle.LATENCY,
+			),
+		[servicename, tagFilterItems],
+	);
+
+	const { isLoading } = useQuery<PayloadProps>({
```

Comment:
what's the sense for this calling this useQuery if we are not consuming this data

Reply:
Yes, agreed. Will remove this.